cxx-qt-gen/src/extract.rs

```diff
@@ -1008,16 +1014,24 @@ pub fn extract_qobject(
                                     _others => original_trait_impls.push(original_impl.to_owned()),
                                 }
                             } else {
-                                let mut extracted = extract_invokables(
-                                    &original_impl.items,
-                                    cpp_namespace_prefix,
-                                    &qt_ident,
-                                )?;
-
-                                object_invokables.append(&mut extracted.invokables);
-                                object_normal_methods.append(&mut extracted.normal_methods);
+                                // Rust only methods are on the impl T block
+                                object_rust_methods.append(&mut original_impl.items);
                             }
                         }
+                        // Invokables are defined in the impl Qt<T> block
+                        "cxx_qt"
+                            if path.segments.len() > 1
+                                && path.segments[1].ident.to_string().as_str() == "QObject" =>
```

Comment:
The premature optimizer monkey in my brain tells me this can probably be "simplified":
`path.segments.get(1).filter(|seg| seg.ident.to_string().as_str() == "QObject").is_some()`
Then we don't need the segments.len() check ;)

Anyway, interesting mental exercise, not important